### DISCUSSION TITLE
Having tab to confirm autocompletion

### DISCUSSION THREAD

**Discussion by loku74** [NvChad/NvChad#2091] [Q&A]
Hello,

I've been trying to use tab to confirm autocompletion. I tried modifying the cmp.lua with this : (I just copied the <CR> function from above)
<img width="455" alt="Capture d’écran 2023-06-06 à 22 30 16" src="https://github.com/NvChad/NvChad/assets/92410060/d4be983c-d859-4b19-a697-0e24a34b8c22">

Without success. Looking for help

**Comment by siduck** [comment]
show your plugin spec
